src/core/event.js

```diff
@@ -482,9 +483,19 @@ Candy.Core.Event = (function(self, Strophe, $) {
 			}
 
 			if(directInvite.length > 0) {
```

Comment:
What's the difference between a mediated invite and a direct invite? Can we make any assumptions about whether the sender is in our global roster with one or the other?